app/models/gids_redis.rb

```diff
@@ -19,6 +19,10 @@ def method_missing(name, *args)
       self.rest_call = name
       self.scrubbed_params = args.first
       response_from_redis_or_service.body
+    elsif search_respond_to?(name)
+      self.rest_call = name
```

Comment:
Since both conditionals assign self.rest_call and self.scrubbed_params, I think the repeated code can be moved out and refactored

Reply:
setting these properties regardless of conditional path

Reply:
changed